packages/amplify_api/example/amplify/backend/function/flutterapiintegtest17270866/src/index.js

```diff
@@ -0,0 +1,15 @@
+
+
+exports.handler = async (event) => {
+    // TODO implement
+    const response = {
+        statusCode: 200,
+    //  Uncomment below to enable CORS requests
+    //  headers: {
+    //      "Access-Control-Allow-Origin": "*",
+    //      "Access-Control-Allow-Headers": "*"
+    //  }, 
+        body: JSON.stringify('Hello from Lambda!'),
```

Comment:
Can you return just the string? I was a bit confused why it was quoted in the test case - think a string makes more sense than a "JSON string"

Reply:
updated. Good suggestion. This is how the CLI does the simple "hello world" lambda by default but agree it's confusing.